content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -189,6 +189,10 @@ echo useSomeLib(new Helper('some text'))
 
 Global Variables however, are resolved at runtime.
 
+This video reviews using resource files from a Shared Library. A link to the example repository used is also provided in the link:https://youtu.be/eV7roTXrEqg[video description].
```

Comment:
```suggestion
This video reviews using resource files from a Shared Library. 
A link to the example repository used is also provided in the link:https://youtu.be/eV7roTXrEqg[video description].
```

Reply:
totally makes sense @Dan-Heath thanks for the heads up!